From 186ac4aee25f89f69fe2bed10aa19478caaca4a6 Mon Sep 17 00:00:00 2001
From: Dan Pasanen <dan.pasanen@gmail.com>
Date: Fri, 29 Dec 2017 11:08:39 -0600
Subject: [PATCH 2/4] init: always allow local.prop overrides

* Many people use this for more than just debugging. Let's allow
  it to work in non-debuggable situations too.

Change-Id: Ifb0d0e636c225ff8830154a49a182a034332056b

Conflicts:
	init/Android.bp
---
 init/Android.bp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/Android.bp b/init/Android.bp
index a2ffed7dc5..f4ae83c3fa 100644
--- a/init/Android.bp
+++ b/init/Android.bp
@@ -31,7 +31,7 @@ cc_defaults {
 //      "-Wno-error=type-limits",
 //      "-Wno-format-extra-args",
         "-DTARGET_PRODUCT=\"$(TARGET_PRODUCT)\"",
-        "-DALLOW_LOCAL_PROP_OVERRIDE=0",
+        "-DALLOW_LOCAL_PROP_OVERRIDE=1",
         "-DALLOW_PERMISSIVE_SELINUX=0",
         "-DREBOOT_BOOTLOADER_ON_PANIC=0",
         "-DWORLD_WRITABLE_KMSG=0",
-- 
2.17.1

